cmatrix.c

```diff
@@ -629,7 +672,7 @@ if (console) {
                 case 'r':
                      rainbow = 1;
                      break;
-                case 'm':
+                case 'M':
```

Comment:
Instead of modifying an existing option, can we switch the new option to use another letter or capital M if no other letters seem fit for the cause?